src/core/core.platform.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+// Core.Platform abstracts away browser specific APIs from the chart
+module.exports = function(Chart) {
+	var platform = Chart.corePlatform = {};
+
+	/**
+	 * @name Core.platform.events
+	 * @type Map<String, String>
+	 */
+	platform.events = {
```

Comment:
What I mean is I don't think we even need variables, just a doc comment with all Chart.js event names, then directly use strings into the code (e.g. `if (e.type === 'mouseenter') { ... }` and in the BrowserPlatform map: `pointerenter: 'mouseenter'`).